src/PVRIptvData.cpp

```diff
@@ -1482,3 +1487,28 @@ int PVRIptvData::GetChannelId(const char * strChannelName, const char * strStrea
 
   return abs(iId);
 }
+
+std::string& PVRIptvData::AddHeaderToStreamUrl(std::string& strLine, const std::string& header)
+{
+  size_t found = strLine.find('|');
+
+  if (found != std::string::npos)
+  {
+    size_t present = strLine.find(header + '=', found + 1);
+
+    if (present == std::string::npos)
+    {
```

Comment:
You don't need either set of brackets in this if/else.

Reply:
I need the return only be executed when the if it's not verified. Not in both cases. However i can spare some brackets for the "single line" statements.